src/Query/TaskGroups.ts

```diff
@@ -60,13 +72,13 @@ export class TaskGroups {
         return output;
     }
 
-    private addTasks(initialGroups: IntermediateTaskGroups) {
+    private addTasks(taskGroupingTree: TaskGroupingTree) {
```

Comment:
I wonder what the comment may be here. We "add tasks" but rather than that we parse the task grouping tree (very clear now!) and convert it into `TaskGroup`s? Shall the method be renamed to something with this sense?